include/qudarand_kernel.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
```

Comment:
I think this file should be renamed to `quda_rand_kernel.h` to make it more readable.